assets/views/profile/profile-controller.js

```diff
@@ -1,12 +1,13 @@
-angular.module('beamer.controllers.main', [])
+/** profile */
```

Comment:
ignoring this file as discussed, for the next PullRQ try not to add un-finished stuff, i know it is hard to do though bc I try to do it all the time >.<